enarx-keep-sev-shim/src/print.rs

```diff
@@ -68,9 +68,9 @@ macro_rules! print {
 /// Panics if writing to the host fails.
 #[macro_export]
 macro_rules! println {
-    () => (crate::serial_print!("\n"));
-    ($fmt:expr) => (crate::serial_print!(concat!($fmt, "\n")));
-    ($fmt:expr, $($arg:tt)*) => (crate::serial_print!(concat!($fmt, "\n"), $($arg)*));
+    () => (crate::print!("\n"));
+    ($fmt:expr) => (crate::print!(concat!($fmt, "\n")));
+    ($fmt:expr, $($arg:tt)*) => (crate::print!(concat!($fmt, "\n"), $($arg)*));
```

Comment:
`$crate` instead is probably more correct.

Reply:
nice spot!

Reply:
fixed